raiden/network/transport/matrix/client.py

```diff
@@ -696,13 +693,6 @@ def _handle_message(
         while True:
             gevent.wait({response_queue, stop_event}, count=1)
 
-            if stop_event.is_set():
```

Comment:
Hummm, why did you move this code bellow?

Reply:
I moved it there so that we dont run into the case where we have a lot of responses in the queue and handle all of them before getting back to the condition. This was caught before by should listen. The `if` in that position replaces it and renders the old place (where it was) useless.